src/content/docs/fr/guides/integrations-guide/markdoc.mdx

```diff
@@ -0,0 +1,519 @@
+
+Cet exemple rend les guillemets avec un composant personnalisé `Quote.astro` :
+
+```js title="markdoc.config.mjs"
+import { defineMarkdocConfig, nodes, component } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  nodes: {
+    blockquote: {
+      ...nodes.blockquote, // Appliquer les valeurs par défaut de Markdoc pour les autres options
+      render: component('./src/components/Quote.astro'),
+    },
+  },
+});
+```
+
+📚 [Trouver tous les nœuds et attributs de nœuds intégrés à Markdoc dans leur documentation](https://markdoc.dev/docs/nodes#built-in-nodes)
+
+### Utiliser des composants d'interface utilisateur côté client
+
+Les balises et les nœuds sont limités aux fichiers `.astro`. Pour intégrer des composants d'interface utilisateur côté client dans Markdoc, [utilisez un composant `.astro` enveloppant qui rend un composant de framework](/fr/core-concepts/framework-components/#imbriquer-des-composants-de-framework) avec la directive `client:` de votre choix.
+
+Cet exemple englobe un composant React `Aside.tsx` avec un composant `ClientAside.astro` :
+
+```astro title="src/components/ClientAside.astro"
+---
+import Aside from './Aside';
+---
+
+<Aside {...Astro.props} client:load />
+```
+
+Ce composant Astro peut maintenant être passé à la propriété `render` pour n'importe quel [tag][markdoc-tags] ou [node][markdoc-nodes] dans votre configuration :
+
+```js title="markdoc.config.mjs"
+import { defineMarkdocConfig, component } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  tags: {
+    aside: {
+      render: component('./src/components/ClientAside.astro'),
+      attributes: {
+        type: { type: String },
+      },
+    },
+  },
+});
+```
+
+### Markdoc config
+
+Le fichier `markdoc.config.mjs|ts` accepte [toutes les options de configuration de Markdoc](https://markdoc.dev/docs/config), y compris [les balises](https://markdoc.dev/docs/tags) et [les fonctions](https://markdoc.dev/docs/functions).
+
+Vous pouvez passer ces options à partir de l'exportation par défaut dans votre fichier `markdoc.config.mjs|ts` :
+
+```js title="markdoc.config.mjs"
+import { defineMarkdocConfig } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  functions: {
+    getCountryEmoji: {
+      transform(parameters) {
+        const [country] = Object.values(parameters);
+        const countryToEmojiMap = {
+          japan: '🇯🇵',
+          spain: '🇪🇸',
+          france: '🇫🇷',
+        };
+        return countryToEmojiMap[country] ?? '🏳';
+      },
+    },
+  },
+});
+```
+
+Vous pouvez désormais appeler cette fonction à partir de n'importe quelle entrée de contenu Markdoc :
+
+```md
+¡Hola {% getCountryEmoji("spain") %}!
```

Comment:
```suggestion
¡Bonjour {% getCountryEmoji("france") %}!
```

Reply:
J'ai préféré laisser ça comme tel comme un exemple, montrer qu'on peut le faire dans une autre langue. Tu penses tout de même à le mettre en français ?

Reply:
Pas de problème ca me va.

Reply:
I believe it remains as `"spain"` for the other translations, as well.